lib/omniauth/strategies/oidc/back_channel_logout_request.rb

```diff
@@ -67,14 +74,27 @@ def sign_out_callback
           @options[:enable_remote_sign_out]
         end
 
-        def backchannel_logout_response(code, body)
+        def back_channel_logout_response(code, body)
           response = Rack::Response.new
           response.status = code
           response['Cache-Control'] = 'no-cache, no-store'
           response['Pragma'] = 'no-cache'
+          response.headers['Content-Type'] = 'text/plain; charset=utf-8'
           response.body = body
           response
         end
+
+        def issuer
+          @options.issuer.nil? ? 'https://connect.authentiq.io/':@options.issuer
+        end
+
+        def algorithm
+          if @options.algorithm != nil && (%w(HS256 RS256 ES256).include? @options.client_signed_response_alg)
```

Comment:
Not 100% sure what this does, but I don't think this module supports `RS*` or `ES*` (without significant additions).

What's the difference between `@options.algorithm` and `@options.client_signed_response_alg`?